### DISCUSSION TITLE
Trying to install enhanced-table-1.13.3_osd-2.2.0.zip in opensearch dashboards 2.2.0 but getting error.

### DISCUSSION THREAD

**Discussion by Rajeshdaneti** [fbaligand/kibana-enhanced-table#298] [General]
Hi All,
Trying to install enhanced-table-1.13.3_osd-2.2.0.zip in opensearch dashboards 2.2.0 but getting following error-
Please assist me in solving the problem.

[opensearch-dashboards@opensearch-dashboards-86495f9499-6c8jj ~]$ bin/opensearch-dashboards-plugin install https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.13.3/enhanced-table-1.13.3_osd-2.2.0.zip
Found previous install attempt. Deleting...
Attempting to transfer from https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.13.3/enhanced-table-1.13.3_osd-2.2.0.zip

/usr/share/opensearch-dashboards/src/cli_plugin/lib/log_warnings.js:45
logger.error(warning);
^
TypeError: logger.error is not a function
at process. (/usr/share/opensearch-dashboards/src/cli_plugin/lib/log_warnings.js:45:12)
at process.emit (events.js:400:28)
at process.emit (/usr/share/opensearch-dashboards/node_modules/source-map-support/source-map-support.js:516:21)
at doEmitWarning (internal/process/warning.js:70:11)
at processTicksAndRejections (internal/process/task_queues.js:81:21)

**Comment by fbaligand** [comment]
Hi,
I just managed to install enhanced-table-1.13.3_osd-2.2.0.zip  on OSD 2.2.0.
My logs:
```
[opensearch-dashboards@23593d071fae bin]$ ./opensearch-dashboards-plugin install https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.13.3/enhanced-table-1.13.3_osd-2.2.0.zip
Attempting to transfer from https://github.com/fbaligand/kibana-enhanced-table/releases/download/v1.13.3/enhanced-table-1.13.3_osd-2.2.0.zip
Transferring 493136 bytes....................
Transfer complete
Retrieving metadata from plugin archive
Extracting plugin archive
Extraction complete
Plugin installation complete
```

Given your logs, it seems that plugin download fails.
Maybe you could not access internet from your OSD machine.
You could try to download enhanced table plugin on another machine (connected to internet), then transfer the package to OSD machine, and finally execute:
`bin/opensearch-dashboards-plugin install file:///path/to/enhanced-table-1.13.3_osd-2.2.0.zip`

Another thing that I see in your logs: ` Found previous install attempt. Deleting...`
Before installing the plugin, you could try to remove `plugins/enhancedTable` directory and eventually hidden files/directories in `plugins`, I mean files/directories which name starts with a "."

**Reply by Rajeshdaneti** [reply]
Thank you very much for your support. Now it is working fine, issue was with firewall which was not allowing to access the link.

Regards
Rajesh Kumar

**Reply by fbaligand** [reply]
Nice to see that it finally works!
